engine/pong.cpp

```diff
@@ -81,9 +91,9 @@ void Pong::Cleanup()
     GetInstance().mGameObjects.clear();
 }
 
-void Pong::SetTimeout(int gameObjectId, int timeoutMs, std::function<void()> callback)
+void Pong::SetTimeout(int gameObjectId, std::chrono::duration<double> timeout, std::function<void()> callback)
```

Comment:
You could take these by const reference i would guess. I don't think std::function it makes much of a difference, but the chrono object it is probably preferable